docs/book/src/reference/metrics.md

```diff
@@ -73,6 +76,29 @@ for the metrics exported from the namespace where the project is running
 
 <img width="1680" alt="Screenshot 2019-10-02 at 13 07 13" src="https://user-images.githubusercontent.com/7708031/66042888-a497da80-e515-11e9-9d77-d8a9fc1159a5.png">
 
+## Default Metrics References
+
+By default, [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) provides controller-relative metrics in different perspectives:
+
+| Metrics name                                  | Type      | Description                                                                                                                                                                                                                          |
```

Comment:
Are we not also expecting by default: https://github.com/2uasimojo/controller-runtime/blob/master/pkg/metrics/client_go_adapter.go?